charts/rancher-backup/values.yaml

```diff
@@ -90,3 +90,19 @@ monitoring:
     additionalLabels: {}
     metricRelabelings: []
     relabelings: []
+  
+  alerts:
+    enabled: false
+
+    ## Define custom alerting rules here.
+    ## The "BackupFailed" alert is included by default when .Values.monitoring.alerts.enabled is set to true and rancher-monitoring is installed.
+    alertingRules:
+    - alert: BackupFailed
```

Comment:
Nice.

Only thing I'm wondering about is, "are there any alerts that BRO should provide by default that cannot be disabled". In other words, are there any features of this new monitoring/alerting area of BRO that (when enabled) are required? I guess this thought goes just as much for these alerting rules as it does for any metrics collection things too - since I could imagine this could relate more to dashboards than alerts TBH. 🤔